servicecontrol/monitoring-instances/installation/creating-config-file.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: Monitoring Instance Configuration Settings
-summary: Categorized list of ServiceControl Monitoring instance configuration settings.
-reviewed: 2021-04-30
+summary: Categorized ServiceControl Monitoring instance configuration settings list.
+reviewed: 2024-05-07
 ---
 
 
 ## Configuration Settings
 
-The configuration of a Monitoring instance can be adjusted via ServiceControl Management or by directly modifying the `ServiceControl.Monitoring.exe.config` file. The settings listed are applicable to the app settings section of the configuration file unless otherwise specified.
+The configuration of a Monitoring instance can be adjusted via ServiceControl Management or by directly modifying the `ServiceControl.Monitoring.exe.config` file. Unless otherwise specified, the settings listed apply to the app settings section of the configuration file.
```

Comment:
```suggestion
The Monitoring instance configuration can be adjusted via ServiceControl Management or by directly modifying the `ServiceControl.Monitoring.exe.config` file. Unless otherwise specified, the settings listed apply to the app settings section of the configuration file.
```